### ISSUE TITLE
how to use Logistic and CrossEntropy loss Function for C++

### ISSUE THREAD

**Issue by Eomtaesan** [Microsoft/CNTK#2781]
I think there is no loss function Logistic and CrossEntropy (without softmax).
how to use it?

**Comment by faruknane** [comment]
I have been searching for cross-entropy without softmax. I couldn't find yet

**Comment by getmarked** [comment]
Why would you want cross entropy without softmax?

**Comment by delzac** [comment]
Why do you need cross entropy without softmax?

**Comment by faruknane** [comment]
> Why do you need cross entropy without softmax?

I just don't get it why cntk prefers not putting softmax layer into the models. Normally that's what we do. Because there is no cross entropy without softmax, we have to remove the softmax layers from our models.

**Comment by delzac** [comment]
Its not just cntk. Every major deep learning framework does that.

In inference, you do not require softmax in the last layer to determine which class is the most probable.
